tests/e2e-tests/.env.template

```diff
@@ -1,9 +1,9 @@
 process.env.AUDIT_BUCKET_NAME = 'audit-build-message-batch-txma2'
 process.env.NOTIFY_API_KEY = 'NOTIFY_API_KEY'
 process.env.TEST_DATA_BUCKET_NAME = 'txma-audit-dev-tools-build-test-data-bucket'
-process.env.ZENDESK_ADMIN_EMAIL = 'txma-team2-ticf-admin-dev@test.gov.uk'
-process.env.ZENDESK_AGENT_EMAIL = 'txma-team2-ticf-approver-dev@test.gov.uk'
+process.env.ZENDESK_ADMIN_EMAIL = 'txma-team2-ticf-admin-dev@example.com'
```

Comment:
these ok to have their domain changed from gov.uk to .com?

Reply:
have updated quite a few of the test emails back to gov.uk - more appropriate

Reply:
can we avoid changing these as these are part of the tests